package.json

```diff
@@ -36,6 +36,7 @@
     "@kth/session": "^3.0.5",
     "@opentelemetry/api": "^1.1.0",
     "@opentelemetry/sdk-trace-base": "^1.6.0",
+    "@testing-library/user-event": "^14.4.3",
```

Comment:
This dependency is used for test/local/pipeline but not for production code. Therefore it should be moved from `dependencies` to `devDependencies` list in package.json